jenkins/scripts/bml_integration_test.sh

```diff
@@ -69,8 +69,8 @@ ssh \
   -o ServerAliveCountMax=10 \
   -i "${AIRSHIP_CI_USER_KEY}" \
   "${AIRSHIP_CI_USER}"@"${TEST_EXECUTER_IP}" \
-  -o SendEnv="MYVAR" \
-  -o SendEnv="MYVAR" \
+  -o SendEnv="BML_ILO_USERNAME=${BML_ILO_USERNAME}" \
```

Comment:
I don't think you need the `=` and right hand side. The value of BML_ILO_USERNAME is already set in the environment where this is run.